movingSquare.js

```diff
@@ -0,0 +1,93 @@
+const square = document.createElement('div')
```

Comment:
App contains a bug. Square go 1px too far and make viewport scroll on Y axis.

<img width="1440" alt="Screen Shot 2022-05-26 at 14 35 28" src="https://user-images.githubusercontent.com/20150525/170488885-a18a9c2e-3f93-44a9-b455-0034498de9fc.png">